python/ray/tests/conftest.py

```diff
@@ -376,6 +376,21 @@ def ray_start_with_dashboard(request, maybe_external_redis):
         yield address_info
 
 
+@pytest.fixture
+def ray_start_dashboard_bad_import():
+    import sys
```

Comment:
do we need to do ray init here?

Reply:
We want the init to fail, so we do it in the test